docker-app/qfieldcloud/core/models.py

```diff
@@ -1186,21 +1186,32 @@ def problems(self) -> list[dict]:
                 layer_name = layer_data.get("name")
 
                 if layer_data.get("error_code") != "no_error":
+                    if layer_data.get("error_code") == "localized_dataprovider":
+                        description = _('Layer "{}" dataprovider is localized').format(
+                            layer_name
+                        )
+                        solution = _(
+                            'Make sure your <a href="https://docs.qfield.org/fr/how-to/outside-layers/">localized layer</a> is available on your QField device.'
+                        )
+                    else:
+                        description = _(
+                            'Layer "{}" has an error with code "{}": {}'
+                        ).format(
+                            layer_name,
+                            layer_data.get("error_code"),
+                            layer_data.get("error_summary"),
+                        )
+                        solution = _(
+                            'Check the last "process_projectfile" logs for more info and reupload the project files with the required changes.'
```

Comment:
And not related to PR changes:

we could here add the link with `project_jobs` url ... if we are already on it ?

Reply:
We cannot really, because we don't have the web UI here.